heartbeat/galera

```diff
@@ -955,7 +965,7 @@ fi
 : ${OCF_RESKEY_check_user="root"}
 
 MYSQL_OPTIONS_CHECK="-nNE --user=${OCF_RESKEY_check_user}"
-if [ -n "${OCF_RESKEY_check_passwd}" ]; then
+if [ -n "${OCF_RESKEY_check_passwd}" ] || [ [ -z "${OCF_RESKEY_check_passwd}" ] && [ "${OCF_RESKEY_check_passwd_use_empty}" = 1 ] ]; then
```

Comment:
The shellcheck run fails on this with these errors:

```
heartbeat/galera:969:45: error: If grouping expressions inside [..], use \( ..\). [SC1026]
heartbeat/galera:969:47: warning: Unknown binary operator. [SC2057]
```
Looking at it, it looks like you are nesting `[ [ ... ]  && [ ... ] ]` - that looks off to me.

Reply:
urgs. fixed. thanks